go/test/endtoend/messaging/message_test.go

```diff
@@ -274,6 +271,75 @@ func TestThreeColMessage(t *testing.T) {
 	assert.Equal(t, uint64(1), qr.RowsAffected)
 }
 
+var createSpecificStreamingColsMessage = `create table vitess_message4(
```

Comment:
Any reasons not to use the new structure we have in the docs? Not a big deal, but I'd like to avoid drift between what we test and what we tell users to use if possible from here on out.

Reply:
Just didn't think to change it. I'll update the tests

Reply:
done